### PR TITLE
implement per-thread, per-batch sleep in ms (throttling allows easier UI dev)

### PR DESCRIPTION
**Pull request by sunnygleason**
#### Problem

* when building a UI to visualize ledger updates, the current client.sh produces a ton of traffic since it's designed to max out TPS
* it would be nice to be able to reduce the number of threads *and* introduce a sleep between iterations to provide more control, resulting in a more manageable TPS

#### Summary of Changes

* add a command-line argument to the bench-tps program for per-thread batch sleep ms
* defaults to zero (which is interpreted as "no sleep")
* plumb the command-line argument into the per-thread transaction processing loop

Fixes #

N/A

### CODE REVIEW

File: bench-tps/src/bench.rs [hunk 1]
```diff
@@ -233,9 +233,13 @@ pub fn do_tx_transfers(
     leader: &NodeInfo,
     shared_tx_thread_count: &Arc<AtomicIsize>,
     total_tx_sent_count: &Arc<AtomicUsize>,
+    thread_batch_sleep_ms: usize,
 ) {
     let client = mk_client(&leader);
     loop {
+        if thread_batch_sleep_ms > 0 {
+            sleep(Duration::new(0, thread_batch_sleep_ms as u32 * 1_000_000));
```
Comment (sakridge) [inline bench-tps/src/bench.rs]: `Duration::from_ms(thread_batch_sleep_ms)` should be a bit cleaner here.

Comment (sunnygleason) [reply bench-tps/src/bench.rs]: @sakridge fixed, thank you! (from a rust newbie)

**Comment by mvines** [comment]
Running `cargo fmt` will get the "checks" build green

File: multinode-demo/client-throttled.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```
Comment (mvines) [inline multinode-demo/client-throttled.sh]: I'm not super keen on growing `multinode-demo/`.   I'd rather whoever will run `multinode-demo/client-throttled.sh` instead just runs `multinode-demo/client.sh` and supplies the different arguments

Comment (sunnygleason) [reply multinode-demo/client-throttled.sh]: 💯

**Pull request merged**